history.md

```diff
@@ -0,0 +1,3 @@
+Currently, every backup that is taken is called exactly the same. The outcome of this is that the newest backup always overwrites the previous one. ReaR, however, allows the usage of the configuration parameter ISO_PREFIX to change the name of the generated ISO. 
```

Comment:
I understood the meaning of it, but I think this should be more high level description.
Something like:
The backup and restore feature currently does not set anything automatically to have several backups without overwritten them. (Have a historical list of the backups). The purpose of this is to configure ReaR to have them automatically.

Managing the historical backup list is not the purpose of this RFE.

Reply:
Please, contribute to the spec with the ideas you gathered through the analysis you did while implementing this RFE.